Sources/BluetoothServices/Characteristics/PnPID.swift

```diff
@@ -0,0 +1,128 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import NIO
+import SpeziBluetooth
+
+
+/// Bluetooth Vendor ID Source.
+public enum VendorIDSource {
```

Comment:
Would it be possible to link to the documentation detailing the structure of this type so we could easily back reference the reasoning behind the implementation here?